lib/check-dependencies.js

```diff
@@ -126,7 +126,7 @@ async function checkForDependencies (opts = {}) {
 }
 
 async function bundleWDASim (xcodebuild, opts = {}) {
-  if (!_.isFunction(xcodebuild?.retrieveDerivedDataPath)) {
+  if (xcodebuild && !_.isFunction(xcodebuild.retrieveDerivedDataPath)) {
```

Comment:
[nits]
Perhaps once you up to date npm modules, gulp can compile `?`

Reply:
the gulp plugins module must be at 5.2.0 or higher

Reply:
probably worth updating that to make this line more readable

Reply:
The scripts run from 'node' directly (for performance sake) so no transpiling. Do you mind if I just leave this as is (I do like this syntax though).